tests/test_using.py

```diff
@@ -2248,6 +2521,29 @@ def incr_one(self, _create, _increment):
         self.assertEqual(3, obj.one)
         self.assertFalse(hasattr(obj, 'incr_one'))
 
+
+    def test_post_generation_async(self):
+        class TestAsyncFactory(FakeAsyncModelFactory):
+            class Meta:
+                model = AsyncTestModel
+
+            one = 1
+
+            @factory.post_generation
+            def incr_one(self, _create, _increment):
+                self.one += 1
+
+        obj = TestAsyncFactory.create()
```

Comment:
This test is incorrect. This is not using async creation.  😅 
That was too good to be true... 😁